de.ient.RDPlot.appdata.xml

```diff
@@ -11,12 +11,12 @@
     <p>RD Plot is a plotting tool for visualizing rate-distortion data parsed from
 encoder log files of common video coding reference software packages.
 It can:
-    Display rate-distortion plots
-    Select component to be plotted (YUV, Y, U, V and even more)
-    Plot rd plot over time (rate over time, psnr over time)
-    Exporting plots (through matplotlib interface)
-    Calculate and display table of bjontegaard delta measurement
-    Export tables as latex style table code
+    <ul>Display rate-distortion plots</ul>
```

Comment:
This isn't valid. For one use `appstream-util validate`.

Anyway its html style syntax:
```html
<p>It supports these features:</p>
<ul>
  <li>Does cool thing #1</li>
  <li>Does cool thing #2</li>
</ul>
```